README.md

```diff
@@ -35,6 +35,7 @@ Install dependencies
 python3 -m venv env
 source env/bin/activate
 pip install -r requirements-min.txt
+export FLASK_DEBUG=1
```

Comment:
I had a bit of trouble running this locally at first. All requests gave me a 30x to redirect me via https, but if I tried to follow the redirect or hit it directly, I got SSL errors. Since I didn't see anything about SSL configuration here, I dug a bit and found the default setting for flask + talisman is to force https. Here, debug mode is used to remove this restriction. As a result, the docker image that also uses this is not production ready